tests/test_webvtt.py

```diff
@@ -393,3 +393,17 @@ def test_save_updated_identifiers(self):
         ]
 
         self.assertListEqual(lines, expected_lines)
+
+    def test_content_formatting(self):
+        """
+        Verify that content property returns the correctly formatted webvtt.
+        """
+        captions = [
+            Caption('00:00:00.500', '00:00:07.000', ['Caption test line 1', 'Caption test line 2']),
+            Caption('00:00:08.000', '00:00:15.000', ['Caption test line 3', 'Caption test line 4']),
+        ]
+        expected_content = "WEBVTT" \
```

Comment:
Can you use textwrap dedent for better readability?
`
textwrap.dedent("""
    WEBVTT
    
    00:00:00.500 --> 00:00:07.000
    Caption test line 1
    Caption test line 2
...
    """).strip()
`